bcda/api/requests_test.go

```diff
@@ -906,7 +906,7 @@ func (s *RequestsTestSuite) TestJobFailedStatus() {
 			rctx.URLParams.Add("jobID", "1")
 
 			ctx := context.WithValue(req.Context(), chi.RouteCtxKey, rctx)
-			ctx = context.WithValue(ctx, logging.CtxTransactionKey, uuid.New())
+			ctx = context.WithValue(ctx, appMiddleware.CtxTransactionKey, uuid.New())
```

Comment:
makes sense - this got moved from the logging package to the middleware package due to a circular dependency (named appMiddleware, since we are overriding some of chi's middleware functionality) so that was a reference change.